dts/bindings/sensor/avia,hx711.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020 George Gkinis
+# Copyright (c) 2021 Jan Gnip
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Avia HX711 sensor
+
+compatible: "avia,hx711"
+
+include: base.yaml
+
+properties:
+  offset:
+    type: int
+    required: false
+    default: 0
```

Comment:
All use of defaults in bindings must be justified in the descriptions. Please read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings and update this PR accordingly.

Reply:
Thanks for your review @mbolivar-nordic.

I cannot think of a justification for **offset, slope** , and **calibration-weight**.

I just want those values to have an initial known value instead of garbage in case the user wants to set them up at runtime and not define them in his overlay file.

Would it be acceptable to use **DT_INST_PROP_OR** instead? : 
```
#define HX711_INIT(index) \
    const int32_t  def[2]={1,0};                 \
	static struct hx711_data hx711_data_##index = {				      \
		.reading = 0,							      \
		.offset = DT_INST_PROP_OR(index, offset, 0),				      \
		.cw = DT_INST_PROP_OR(index, calibration_weight, &def),			      \
		.sl = DT_INST_PROP_OR(index, slope, &def),				      \
		.sample_fetch_timeout = DT_INST_PROP(index, sample_fetch_timeout_ms), \
		.gain = DT_INST_PROP(index, gain),				      \
		.rate = DT_INST_PROP_OR(index, rate_hz,HX711_RATE_10HZ),				      \
	};									      \
```

For **sample-fetch-timeout-ms** and **gain** there is justification based on the sensor itself: time to first reading and gain after power on.

Reply:
I found a way to justify the default values for offset, slope and calibration weight.

Reply:
@mbolivar-nordic please check if the latest changes are acceptable